game/scripts/npc/items/item_ancient_janggo_of_endurance.txt

```diff
@@ -83,7 +83,7 @@
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_aura_movement_speed"                       "6 10 15 20 25"
+        "bonus_aura_movement_speed"                       "5 8 12 17 20"
```

Comment:
here, it should be "bonus_aura_movement_speed"                       "5 8 12 17 20" //OAA